pkg/volume/util_test.go

```diff
@@ -20,9 +20,10 @@ import (
 	"fmt"
 	"testing"
 
```

Comment:
Since you're fixing these and no good deed goes unpunished-- remove this blank line. :)